gamemodes/PPC_Trucking.pwn

```diff
@@ -139,6 +139,8 @@ public OnPlayerConnect(playerid)
 	if (IsPlayerNPC(playerid))
 		return 1;
 
+   SendDeathMessage(INVALID_PLAYER_ID, playerid, 200); // 200 = DEATHMESSAGE_CONNECT
```

Comment:
Please undo this change as we already have chat messages that inform about players who join / leave the server. If you want file an issue regarding the usage of `SendDeathMessage` in general.

Reply:
Opened issue for this, see #87